index.html

```diff
@@ -1350,55 +1354,82 @@
-            <code>message</code> as <code>closeMessage</code>.
+            with <a for="PresentationConnectionClosedReason">closed</a> as
+            <var>closeReason</var> and an empty message as
+            <var>closeMessage</var>.
           </p>
           <p>
             When the <code><dfn>terminate</dfn>()</code> method is called on a
-            <a>PresentationConnection</a> in a <a>controlling browsing
-            context</a>, the <a>user agent</a> MUST run the algorithm to
-            <a data-lt="terminate-algorithm-controlling">terminate the
-            presentation in a controlling browsing context</a> with
-            <a>PresentationConnection</a>.
+            <a>PresentationConnection</a> <var>S</var> in a <a>controlling
+            browsing context</a>, the <a>user agent</a> MUST run the algorithm
+            to <a data-lt="terminate-algorithm-controlling">terminate the
+            presentation in a controlling browsing context</a> using
+            <var>S</var>.
           </p>
           <p>
             When the <code>terminate()</code> method is called on a
-            <a>PresentationConnection</a> in a <a>receiving browsing
-            context</a>, the <a>user agent</a> MUST run the algorithm to
-            <a data-lt="terminate-algorithm-receiving">terminate the
-            presentation in a receiving browsing context</a> with
-            <a>PresentationConnection</a>.
+            <a>PresentationConnection</a> <var>S</var> in a <a>receiving
+            browsing context</a>, the <a>user agent</a> MUST run the algorithm
+            to <a data-lt="terminate-algorithm-receiving">terminate the
+            presentation in a receiving browsing context</a> using
+            <var>S</var>.
           </p>
           <p>
             When the <code><dfn>send</dfn>()</code> method is called on a
-            <a>PresentationConnection</a> object with a <code>message</code>,
-            the <a>user agent</a> MUST run the algorithm to <a data-lt=
-            "send-algorithm">send a message through a
-            <code>PresentationConnection</code></a>.
+            <a>PresentationConnection</a> <var>S</var>, the <a>user agent</a>
+            MUST run the algorithm to <a>send a message</a> through
+            <var>S</var>.
           </p>
           <p>
             When a <a>PresentationConnection</a> object <var>S</var> is
             discarded (because the document owning it is navigating or is
             closed) while the <a>presentation connection state</a> of
-            <var>S</var> is <code>connecting</code> or <code>connected</code>,
-            the <a>user agent</a> SHOULD <a>start closing the presentation
-            connection</a> <var>S</var> with <code>wentaway</code> as
-            <code>closeReason</code> and an empty <code>closeMessage</code>.
+            <var>S</var> is <a for="PresentationConnectionState">connecting</a>
```

Comment:
The markup that defines "connecting" has a dfn-for="PresentationConnectionState" as well, which effectively creates a namespace for the definition, and thus references to that definition must also specify that namespace.

In turn, the namespace is required in the definition so that ReSpec may connect that definition with the WebIDL that defines the enumeration. It is also needed to avoid collisions with other possible definitions of the same term (e.g. for "connected" as presentation connection state and "connected" as event name)